beacon-chain/core/epoch/precompute/reward_penalty_test.go

```diff
@@ -117,20 +125,21 @@ func TestAttestationDeltaPrecompute(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	attestedIndices := []uint64{55, 1339, 1746, 1811, 1569, 1413}
+	attestedIndices := []uint64{55, 1339, 1746, 1811, 1569}
 	for _, i := range attestedIndices {
 		base, err := epoch.BaseReward(state, i)
 		if err != nil {
 			t.Errorf("Could not get base reward: %v", err)
 		}
+
 		// Base rewards for getting source right
 		wanted := attestedBalance*base/totalBalance +
-			bp.PrevEpochTargetAttesters*base/totalBalance +
-			bp.PrevEpochHeadAttesters*base/totalBalance
+			bp.PrevEpochTargetAttested*base/totalBalance +
+			bp.PrevEpochHeadAttested*base/totalBalance
 		// Base rewards for proposer and attesters working together getting attestation
 		// on chain in the fatest manner
 		proposerReward := base / params.BeaconConfig().ProposerRewardQuotient
-		wanted += (base - proposerReward) * params.BeaconConfig().MinAttestationInclusionDelay
```

Comment:
This params is being reused in a foor loop, can you move it out? Same for the proposer reward one.